src/sass/_Icon.scss

```diff
@@ -57,16 +57,18 @@
 .ms-Icon--AddPhone::before { @include ms-Icon--AddPhone; }
 .ms-Icon--AddTo::before { @include ms-Icon--AddTo; }
 .ms-Icon--Admin::before { @include ms-Icon--Admin; }
-.ms-Icon--AdminALogo::before { @include ms-Icon--AdminALogo; }
```

Comment:
Is `AdminALogo32` a rename of `AdminALogo`, or an all new icon? Same goes for the ones like `AdminCLogo` to `AdminCLogoInverse32`.

Reply:
1. Do you have any recommendations for testing the original against the updated icon set to be able to get clarity for what's a rename vs what's new? Is there something deployed that shows the current icons? I can clone a second repo if nothing else.

2. What is the process for documenting a rename vs a new icon?

Reply:
1. Yeah, you'll need to get the master branch building and then go to the icons page for the documentation site in the repo. I think that's at http://localhost:8080? Then it would just be visually comparing each of those new icons to one of the removed ones, to see if they're the same (renamed) or really are new icons.
2. If they have been renamed, we can keep the old class names *in addition to the new class names* and point them both at the new mixins. See [_Icon.scss](https://github.com/OfficeDev/office-ui-fabric-core/blob/master/src/sass/_Icon.scss#L37) for examples of this.

Reply:
Updated with renamed vs new icons. Let me know if I'm missing anything else. Thanks!